instagram/index.html

```diff
@@ -1,9 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Instagram</title>
-  </head>
-  <body></body>
+<head>
+  <meta charset="UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+  <title>Instagram</title>
+</head>
+<body>
+<nav>
+  <div class="official">
+    <div class="insta"><img src="img/insta.png" alt="instagram"></div>
+    <div class="logo"><img src="img/logo.png" alt="home"></div>
+  </div>
+  <div class="personal">
+    <div class="compass"><img src="img/compass.png" alt="compass"></div>
+    <div class="heart"><img src="img/heart.png" alt="heart"></div>
+    <div class="person"><img src="img/person.png" alt="myinfo"></div>
+  </div>
+</nav>
+<div class="profile">
+  <div class="info">
+    <div class="photo"><img src="img/profile.png" alt="profile"></div>
+    <div class="edit">
+      <div class="nickname">woonjangahn<img src="img/edit.png" alt="edit"></div>
+      <div class="profile-edit">프로필 편집</div>
+    </div>
+  </div>
+  <div class="places">
+    <div class="place">
+      <img src="img/singapore.jpg" alt="singapore">
+    </div>
+  </div>
+</div>
+<div class="articles">
+  <div class="myinfo">
+    <div class="article-count">
+      <div class="name">게시물</div>
+      <div class="count">17</div>
+    </div>
+    <div class="follower-count">
+      <div class="name">팔로워</div>
+      <div class="count">518</div>
+    </div>
+    <div class="follow-count">
+      <div class="name">팔로우</div>
+      <div class="count">236</div>
+    </div>
+  </div>
+  <div class="article-nav">
+    <div><img src="img/calendar.jpg" alt="calendar"></div>
+    <div><img src="img/support.png" alt="support"></div>
+    <div><img src="img/favorite.png" alt="favorite"></div>
+    <div><img src="img/file.png" alt="file"></div>
+  </div>
+  <div class="line">
+    <div class="article-left"><img src="img/canada.jpeg" alt="canada"></div>
+    <div class="article-center"><img src="img/canada.jpeg" alt="canada"></div>
+    <div class="article-right"><img src="img/canada.jpeg" alt="canada"></div>
+  </div>
+  <div class="line">
+    <div class="article-left"><img src="img/canada.jpeg" alt="canada"></div>
+    <div class="article-center"><img src="img/canada.jpeg" alt="canada"></div>
+    <div class="article-right"><img src="img/canada.jpeg" alt="canada"></div>
+  </div>
+  <div class="line">
+    <div class="article-left"><img src="img/canada.jpeg" alt="canada"></div>
+    <div class="article-center"><img src="img/canada.jpeg" alt="canada"></div>
+    <div class="article-right"><img src="img/canada.jpeg" alt="canada"></div>
+  </div>
```

Comment:
게시글 3개가 생길때마다 `div.line` 을 추가하지 않고 3개째마다 자동으로 줄바꿈을 시킬 수 있으면 좋겠네요!
flexbox 관련 속성 중 하나인 `flex-wrap`에 대해 알아보시면 좋을 것 같습니다.
```suggestion
  <div class="articles">
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
    <div class="article"><img src="img/canada.jpeg" alt="canada"></div>
  </div>
```